### ISSUE TITLE
amp-iframes not loading on ios 13.5

### ISSUE THREAD

**Issue by bradleymyers** [ampproject/amphtml#28585]
**Please only file reports about bugs in AMP here.**

- If you have questions about how to use AMP or other general questions about AMP please ask them on Stack Overflow under the AMP HTML tag instead of filing an issue here: http://stackoverflow.com/questions/tagged/amp-html
- If you have questions/issues related to Google Search please ask them in Google's AMP forum instead of filing an issue here: https://goo.gl/utQ1KZ

If you have a bug for AMP please fill in the following template. Delete everything except the headers (including this text).

## What's the issue?

My iframes aren't loading on ios 13.5 chrome, safari, brave. Non of them load. Work fine on win 10 and Mac OS 10.3.6

## How do we reproduce the issue?

On an iphone running ios13.5 Go to www.bajafresh.com/locator. Map doesn't load. And then in the hamburger menu if you click it and select locations. A form is supposed to load in. Doesn't load. Both instances are delivered via amp-iframe

## What browsers are affected?
Safari iOS, Chrome iOS, Brave iOS

## Which AMP version is affected?

<script async="" src="https://cdn.ampproject.org/v0.js"></script>

**Comment by Ichigo3d** [comment]
Hello. Thank you for participation in improving our product. I checked it on current production 2005151844001 and beta 2005150002000. Browsers: Chrome and Safari. IOS 13.5. But bug is unreproducible. Could you provide screenshots and more info?

**Comment by bradleymyers** [comment]
Thanks for getting back. I was able to fix the issue yesterday evening. There where two problems and only one of them really had anything to do with amp-iframe issue. 

1. On the map iframe there was an SSL issue. Seems that iOS 13.5 didn't like the cert. So I regenerated it and it started working
2. The main nav locations drop down on mobile had an iframe in it. Seems like the height and width of the iframe was to large. Amp seemed to think it was supposed to be an ad. At least that is what the error was throwing. In other browsers its still rendered fine but in iOS 13.5 browsers it wouldn't. Before the iOS update it was working fine so it seems like what ever they did in the release set amp off.

Again, thanks for getting back. Closing out.
